packages/@okta/vuepress-theme-prose/global-components/EventTypes.vue

```diff
@@ -81,9 +81,15 @@
           :key="tag"
           class="event-type-tag"
         >{{ tag }}</code>
+        <code
+          v-if="eventType.schema.changeDetails !== null && (!Array.isArray(eventType.schema.changeDetails) || eventType.schema.changeDetails.length > 0) "
+          :key="tag"
+          class="event-type-tag"
+        >{{ "changeDetails" }}</code>
       </div>
       <div class="event-type-release">
-        Since: <SmartLink :item="{link: '/docs/release-notes/', text: eventType.info.release}" />
+        <!-- Since: <SmartLink :item="{link: '/docs/release-notes/', text: eventType.info.release}" /> -->
+        Since: {{ eventType.info.release }}
```

Comment:
Is this intended change? We'll have no link just a release date. If so, could you remove a commented smart link?

Reply:
Yes, the link was not useful and went to top of Release Notes page; the smartlink cut-off a portion of the release date on right and left side. 

Removed the comment text, thanks.